modin/pandas/series.py

```diff
@@ -42,8 +36,69 @@
 from . import _update_engine
 
 
-@_inherit_docstrings(pandas.Series, excluded=[pandas.Series.__init__])
+@_inherit_docstrings(
+    pandas.Series, excluded=[pandas.Series.__init__], apilink="pandas.Series"
+)
 class Series(BasePandasDataset):
+    """
+    Modin distributed representation of `pandas.Series`.
+
+    Internally passed data is divided into partitions in order to parallelize
+    computations and utilize the user's hardware as much as possible.
+
+    Inherit common for DataFrames and Series functionality from the
+    `BasePandasDataset` class.
+
+    Parameters
+    ----------
+    data : modin.pandas.Series, array-like, Iterable, dict, or scalar value, optional
+        Contains data stored in Series. If data is a dict, argument order is
+        maintained.
```

Comment:
I'm in doubt about maintaining the order when `data` is a `dict`, as normal dictionaries in Python do not guarantee to be ordered.

Reply:
Python `dict` seems to be [ordered](https://docs.python.org/3.7/tutorial/datastructures.html#dictionaries) since python 3.7.